appveyor.yml

```diff
@@ -26,7 +26,7 @@ build_script:
     - if "%configuration%"=="Unicode Release" set scintilla_debug=
     - nmake NOBOOST=1 %scintilla_debug% -f scintilla.mak
     - cd c:\projects\notepad-plus-plus\PowerEditor\visual.net\
-    - msbuild notepadPlus.vcxproj /p:configuration="%configuration%" /p:platform="%platform%"
+    - msbuild notepadPlus.vcxproj /p:configuration="%configuration%" /p:platform="%platform%" /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
```

Comment:
additionally /p:PlatformToolset="%PlatformToolset%" could be added to add the ability to build for v120_xp and v140_xp

via:

environment:
  matrix:
    - PlatformToolset: v120_xp
    - PlatformToolset: v140_xp